Prowl.Editor/EditorApplication.cs

```diff
@@ -49,6 +49,10 @@ public static void SaveConfig()
     {
         string filePath = Path.Combine(Project.Projects_Directory, "EditorConfig.setting");
         string json = JsonSerializer.Serialize(EditorConfig);
+
+        if(!Directory.Exists(Project.Projects_Directory)) 
```

Comment:
CreateDirectory itself already does an Exists check before creating so don't need the if check.